package.json

```
@@ -2,6 +2,9 @@
   "name": "rapid-experiments-shared",
   "version": "0.0.1",
   "license": "MPL-2.0",
+  "engines": {
+    "node": "14"
```

probably `>=14`, right?

---

We had problems on NDT where CI updated to a newer Node version and broke our deploys. Given that I don't know what Node 16 will hold, and if we'll be compatible, I think just 14 is a better idea for now. (Since this says any `14.x` version is compatible). I suppose I could make it more explicit and spell this as `^14.0.0` if you'd prefer?

---

Fair enough, yeah I like `^14.0.0`

---

It's easier to use the suggestion UI than editing the file locally.

```suggestion
    "node": "^14.0.0"
```